tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -90,4 +90,163 @@ TestAssert "test", not ("4,9" is not within "-5,-10,5,10")
 TestAssert "test", "-6,-11" is not within "-5,-10,5,10"
 TestAssert "test", "5,10" is not within "-5,-10,5,10"
 
-end TestGraphicsIsNotWithin
\ No newline at end of file
+end TestGraphicsIsNotWithin
+
+-- new brush mapping for 101..135
```

Comment:
The brushes stack has an image ID 136. In this instance the docs are wrong which of course means we need to ensure all the docs say the correct range.  The following from your PR actually has the range 101-136.

    MCspray > PI_BRUSHES && MCspray <= PI_PATTERNS